lib/fetch/index.js

```diff
@@ -1562,78 +1556,155 @@ function httpNetworkFetch (
 
           context.on('terminated', onResponseAborted)
 
+          const codings =
+            headers
+              .get('content-encoding')
+              ?.toLowerCase()
+              .split(',')
+              .map((x) => x.trim()) ?? []
+
+          const decoders = []
+
+          // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding
+          for (const coding of codings) {
+            if (/(x-)?gzip/.test(coding)) {
+              decoders.push(zlib.createGunzip())
+            } else if (/(x-)?deflate/.test(coding)) {
+              decoders.push(zlib.createInflate())
```

Comment:
Shouldn't this be `zlib.createDeflate()`?

Reply:
inflate is actually correct